tests/infer/mcmc/test_mcmc.py

```diff
@@ -51,6 +51,16 @@ def test_mcmc_interface():
     assert_equal(sample_std, torch.tensor([1.0]), prec=0.08)
 
 
+@pytest.mark.parametrize("num_chains", [1, 2])
```

Comment:
Nice observation! That's why I don't get any error in Travis with last PR. I'll make pytest marks here and open an issue for the error which we caught in this PR.

Reply:
@fehiepsi - Yes, for as long as the test passes locally, it is fine. You can add a skipif marker here for travis. Btw..this isn't a bug, so no issue is required, it is just a check within the MCMC class itself so that we don't dispatch to multiple chains and become slower. Since travis machines do not have multiple cores, it is fine to do any testing for multiple chains locally. We we have a real need for testing parallel chains in travis, we can add additional arguments to force dispatch even when `num_chains > num_cpus` later.

Reply:
Agree, we don't need to test multi-chain in Travis.